docs/rules/multiline-ternary.md

```diff
@@ -136,6 +136,62 @@ foo > bar ? value1 : value2;
 foo > bar ? (baz > qux ? value1 : value2) : value3;
 ```
 
+### --fix
+
+If this rule is invoked with the command-line `--fix` option, it's recommended to define both `indent` and `operator-linebreak` if you want to have sensible results when using the `always` and `always-multiline` options.
```

Comment:
I'm not sure if this documentation update is necessary.

The same applies to a lot of fixable rules, but we usually don't have these sections since it's reasonable to expect that other formatting rules are also enabled and configured. `operator-linebreak` is already listed in "Related Rules".

This detailed explanation with examples is certainly useful, but it's also an additional thing to maintain and keep in sync with other rules.

Thoughts from the team?